repos/newtonsoft-json.proj

```diff
@@ -1,20 +1,27 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <PreferredHash>e43dae94c26f0c30e9095327a3a9eac87193923d</PreferredHash>
+    <PreferredHash>cac0690ad133c5e166ce5642dc71175791404fad</PreferredHash>
     <SourceDirectory>Newtonsoft.Json</SourceDirectory>
   </PropertyGroup>
 
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
+  
+  <!--The Library Frameworks was added to environment variables so as to override the frameworks in newtonsoft csproj-->
+  <!--The semicolons were added with encoding so as to avoid conflicts in MSBuild shell-->
+  <ItemGroup>
+    <EnvironmentVariables Include="LibraryFrameworks=netstandard1.0%3Bnetstandard1.3%3Bnetstandard2.0" />
+    <RepositoryReference Include="arcade" />
```

Comment:
Please comment about how this is a fake RepositoryReference only used for build ordering.